LocalisationTable.cs

```diff
@@ -15,18 +14,31 @@ public class LocalisationTable : ScriptableObject, ISerializationCallbackReceive
 		private string[] locales = {};
 
 		[SerializeField]
-		private List<int> keys = new List<int>();
+		private int crcEncodingVersion;
 
 		[SerializeField]
-		private List<string> values = new List<string>();
+		private List<int> keys = new List<int>();
 
 		[SerializeField]
-		public int crcEncodingVersion;
+		private List<string> values = new List<string>();
```

Comment:
This is a new private property but there is no new code that refers to it.

Reply:
I think all I did was re-order the fields, I don't see a new private property?

Reply:
Ah yes, they are just re ordered.